pkg/client/fleetmanager/client.go

```diff
@@ -0,0 +1,130 @@
+package fleetmanager
+
+import (
+	"net/http"
+	"net/url"
+
+	"github.com/pkg/errors"
+	admin "github.com/stackrox/acs-fleet-manager/internal/dinosaur/pkg/api/admin/private"
+	"github.com/stackrox/acs-fleet-manager/internal/dinosaur/pkg/api/private"
+	"github.com/stackrox/acs-fleet-manager/internal/dinosaur/pkg/api/public"
+)
+
+type authTransport struct {
+	transport http.RoundTripper
+	auth      Auth
+}
+
+func (c *authTransport) RoundTrip(req *http.Request) (*http.Response, error) {
+	if err := c.auth.AddAuth(req); err != nil {
+		return nil, errors.Wrapf(err, "setting auth on req %+v", req)
+	}
+	resp, err := c.transport.RoundTrip(req)
+	if err != nil {
+		return resp, errors.Wrap(err, "round trip error")
```

Comment:
```suggestion
		return resp, err
```
The wrap does not provide any value whatsoever. If the linter insists on it, it needs to be suppressed at this point

Reply:
I guess you can change the entire thing to `return c.transport.RoundTrip(req)`